frontend/src/lib/lemon-ui/LemonTable/LemonTable.scss

```diff
@@ -74,6 +78,16 @@
         max-width: 30rem;
         font-size: 0.75rem;
     }
+
+    a.Link {
```

Comment:
Is it just tables we want to change the colour of the links in?

Reply:
We don't want orange links everywhere, but definitely don't want them here. Rather than making _all_ links the text color, it was safer to do it on a case-by-case basis.